pkg/config/images.go

```diff
@@ -149,6 +149,9 @@ func (g *simpleGenerator) selectContainerImagesOSA(cs *api.OpenShiftManagedClust
 
 		c.Images.Sync = "quay.io/openshift-on-azure/sync:v3.11"
 		c.Images.AzureControllers = "quay.io/openshift-on-azure/azure-controllers:v3.11"
+
+		c.Images.GenevaLogging = "osarpint.azurecr.io/acs/mdsd:11201801"
```

Comment:
given we have validateConfig(), I'm changing my mind about us having a default.  Why not take this out and simplify the hunk below?